src/clients/FlexbaseClient.Card.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Why are we disabling linting? To overrule weak-typing (`any`)? We should be strong-typing our variables.